test-data/unit/fine-grained-cache-incremental.test

```diff
@@ -0,0 +1,183 @@
+-- Test cases for building caches for fine-grained mode using incremental
+-- builds.
+--
+-- '# num_build_steps: N' specifies how many build steps to do before
```

Comment:
Maybe make more explicit, such as "... how many regular build steps to do without the daemon...".